controllers/util/mw_util.go

```diff
@@ -197,10 +258,29 @@ func (mwu *MWUtil) generateVRGRolesManifestWork(namespace string) (*ocmworkv1.Ma
 		return nil, err
 	}
 
-	manifests := []ocmworkv1.Manifest{*vrgClusterRole, *vrgClusterRoleBinding}
+	pvClusterRole, err := mwu.generatePVClusterRoleManifest()
```

Comment:
We do not create PV MW anymore form the hub post #197 so we can remove the PV roles being rolled out to the clusters.

Reply:
Great!  This means the drplacementcontrol roles manifest count test can also change from a `>=` to `=` 2.